### PR TITLE
Added community mailing lists

### PR DESCRIPTION
**Pull request by mtwo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,16 @@ We are on [gitter](https://gitter.im/open-telemetry/community).
...
 ## Get Involved
 
-There is a lot to do! If you are interested in getting involved, please get in touch via [this simple form](https://forms.gle/HCGGoLRzoPm6o29K6). We are a friendly, collaborative group and look forward to working together.
+There is a lot to do! If you are interested in getting involved, please join the mailing lists and attend the community meetings. If you wish to join one of the special interest groups, you can get in touch via [this simple form](https://forms.gle/HCGGoLRzoPm6o29K6). We are a friendly, collaborative group and look forward to working together.
+
+## Mailing Lists and instant messaging
+
+You should join our public mailing lists, which are where we make announcements and provide regular updates: 
+
+- cncf-opentelemetry-community@lists.cncf.io
```
Comment (SergeyKanzhelev) [inline README.md]: can you copy description of the groups from the document, please. Also in the end of document - mention groups for governance and public messaging. So everybody can reach out privately if needed

Comment (mtwo) [reply README.md]: Will do

File: README.md [hunk 2]
```diff
@@ -8,7 +8,16 @@ We are on [gitter](https://gitter.im/open-telemetry/community).
...
+- cncf-opentelemetry-comms@lists.cncf.io
+
+You can sign up for these lists at https://groups.io/groups
```
Comment (pavolloffay) [inline README.md]: I haven't find it there. cncf-opentelemetry-community@lists.cncf.io

Comment (SergeyKanzhelev) [reply README.md]: Links would be like this:  https://lists.cncf.io/g/cncf-opentelemetry-community

@mtwo is working on adding these direct links to the PR

**Comment by mtwo** [comment]
@SergeyKanzhelev updated the PR to reflect your comments.

I also added myself to the public communication list ... who do I need approval from to accomplish this?

**Pull request merged**